draft-fossati-tls-attestation.md

```diff
@@ -1101,6 +1082,17 @@ the TLS server will terminate the exchange.
 
 TBD.
 
+## Security Guarantees {#sec-guarantees}
+
+We note that as a pure cryptographic protocol, attested TLS as-is only guarantees that the Identity Key is known by the TEE. A number of additional guarantees must be provided by the platform and/or the TLS stack,
+and the overall security level depends on their existence and quality of assurance:
+
+* The Identity Key is generated by the TEE.
+* The Identity Key is never exported or leaked outside the TEE.
+* The TLS protocol, whether implemented by the TEE or a layer above it, is implemented correctly and (for example) does not leak any session key material.
+
+These properties may be explicitly promised ("attested") by the platform, or they can be assured in other ways such as by providing source code, reproducible builds, formal verification etc. The exact mechanisms are out of scope of this document.
```

Comment:
A couple of missing commas

```suggestion
These properties may be explicitly promised ("attested") by the platform, or they can be assured in other ways, such as by providing source code, reproducible builds, formal verification, etc. The exact mechanisms are out of scope of this document.
```

Reply:
Some languages are more strict about commas. My personal tendency is similar, but when writing English I adopt the more relaxed American approach.